lib/ecto/query.ex

```diff
@@ -476,16 +476,20 @@ defmodule Ecto.Query do
   than one where expression, they are combined with an `and` operator. All
   where expressions have to evaluate to a boolean value.
 
+  `where` also accepts a keyword list
```

Comment:
What about this for docs:

> `where` also accepts a keyword list where the field given as key is going to be compared with the given value. The fields will always refer to the source given in `from`.
> 
> It is also possible to interpolate the whole keyword list, allowing developers to dynamically filter the source:
>
>     filters = [state: "Sweden"]
>     from(c in City, where: ^filters)

It would be nice to add similar notes to `order_by`, which also supports dynamic fields. :)